404.html

```diff
@@ -0,0 +1,107 @@
+      </a>
+    </div>
+    <script>
+      (function() {
+        'use strict';
+        const redirectUrl = function(url) {
+          var uri = new URL(url);
+          var pattern = /^\/google-http-java-client\/releases\/(\d+\.\d+\.\d+|latest)\/javadoc\/(.*)/;
+          var match = pattern.exec(uri.pathname);
+          if (match == null) {
+            return null;
+          }
+          return 'https://googleapis.dev/java/google-http-client/' + match[1] + '/' + match[2] + uri.search + uri.hash;
+        };
+        var destination = redirectUrl(location.href);
+        if (!destination) {
+          destination = 'https://googleapis.dev/java/google-cloud-clients/latest/index.html';
```

Comment:
Can we rely on the default doc here and just use:
https://googleapis.dev/java/google-cloud-clients/latest/